mailit/bin/handleemail.py

```diff
@@ -152,7 +152,9 @@ def instanciate_answer(self, lines):
         msg = email.message_from_string(msgtxt)
         temporary, permanent = all_failures(msg)
 
-        if temporary or permanent:
+        if temporary:
```

Comment:
I'm not sure if in practice this happens with Exim, but `flufl` might return both email addresses with temporary failures and permanent failures, and if there are any in `permanent` it should still register a bounce. So maybe this condition should be `if temporary and not permanent`. (Or switch the cases round so it's `if permanent ... elif temporary ... else ... end`.)